pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -1028,12 +1028,28 @@ func getPlayBookContext(client splcommon.ControllerClient, cr splcommon.MetaObje
 	}
 }
 
+// isBundlePushComplete checks whether the SHC bundle push is complete or still pending
+func (shcPlayBookContext *SHCPlayBookContext) isBundlePushComplete() bool {
+	scopedLog := log.WithName("isBundlePushComplete").WithValues("crName", shcPlayBookContext.cr.GetName(), "namespace", shcPlayBookContext.cr.GetNamespace())
+
+	cmd := fmt.Sprintf("cat %s", shcBundlePushStatusCheckFile)
+	// check the content of the status file
+	stdOut, stdErr, err := shcPlayBookContext.podExecClient.RunPodExecCommand(cmd)
+	if err != nil || stdErr != "" || !strings.Contains(stdOut, shcBundlePushCompleteStr) {
+		scopedLog.Error(err, "checking the status of SHC Bundle Push failed", "stdout", stdOut, "stderr", stdErr)
+		return false
+	}
```

Comment:
As discussed:

* The file should be cleaned up after the push command return contents are written to it & have been checked
* The contents should also be read for any errors too, so that the push can be retried in subsequent reconciles